app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from os import remove, rename, makedirs, path
+
+
+def move_file(command: str) -> None:
+    command = command.split()[1::]
+    if "/" in command:
+        file_path, file_name = command.rsplit("/", 1)
```

Comment:
does it work fine for you? Because I have this problem:
`
AttributeError: 'list' object has no attribute 'split'
`

Reply:
sorry, kind of typo in `command.split()[1::]`